Src/Microsoft.Dynamic/Microsoft.Dynamic.csproj

```diff
@@ -12,433 +12,16 @@
-    <Compile Include="Runtime\ExtraKeyEnumerator.cs" />
-    <Compile Include="Runtime\IMembersList.cs" />
-    <Compile Include="Runtime\ImplicitConversionMethodAttribute.cs" />
-    <Compile Include="Runtime\ModuleChangeEventArgs.cs" />
-    <Compile Include="Runtime\ModuleChangeEventType.cs" />
-    <Compile Include="Runtime\NullTextContentProvider.cs" />
-    <Compile Include="Runtime\OperationFailed.cs" />
-    <Compile Include="Runtime\OperatorSlotAttribute.cs" />
-    <Compile Include="Runtime\PropertyMethodAttribute.cs" />
-    <Compile Include="Runtime\ReflectionCache.cs" />
-    <Compile Include="Runtime\ScriptingRuntimeHelpers.cs" />
-    <Compile Include="Runtime\StaticExtensionMethodAttribute.cs" />
-    <Compile Include="Runtime\Uninitialized.cs" />
-    <Compile Include="Utils\ArrayUtils.cs" />
-    <Compile Include="Utils\AssemblyQualifiedTypeName.cs" />
-    <Compile Include="Utils\Assert.cs" />
-    <Compile Include="Utils\CheckedDictionaryEnumerator.cs" />
-    <Compile Include="Utils\CollectionUtils.cs" />
-    <Compile Include="Utils\ContractUtils.cs" />
-    <Compile Include="Utils\DictionaryUnionEnumerator.cs" />
-    <Compile Include="Utils\ExceptionFactory.Generated.cs" />
-    <Compile Include="Utils\ExceptionUtils.cs" />
-    <Compile Include="Utils\IOUtils.cs" />
-    <Compile Include="Utils\ReflectionUtils.cs" />
-    <Compile Include="Utils\StringUtils.cs" />
-    <Compile Include="IValueEquality.cs" />
-    <Compile Include="KeyboardInterruptException.cs" />
-    <Compile Include="SourceFileContentProvider.cs" />
-    <Compile Include="zlib.net\Adler32.cs" />
-    <Compile Include="zlib.net\Deflate.cs" />
-    <Compile Include="zlib.net\InfBlocks.cs" />
-    <Compile Include="zlib.net\InfCodes.cs" />
-    <Compile Include="zlib.net\Inflate.cs" />
-    <Compile Include="zlib.net\InfTree.cs" />
-    <Compile Include="zlib.net\StaticTree.cs" />
-    <Compile Include="zlib.net\Tree.cs" />
-    <Compile Include="zlib.net\Zlib.cs" />
-    <Compile Include="zlib.net\ZStream.cs" />
-    <Compile Include="zlib.net\ZStreamException.cs" />
+    <Compile Include="..\AssemblyInfo.cs" Link="AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Content Include="Microsoft.Scripting.txt" />
-    <Content Include="zlib.net\history.txt" />
-    <Content Include="zlib.net\license.txt" />
-    <Content Include="zlib.net\readme.txt" />
-    <None Include="Microsoft.Dynamic.ruleset" />
-  </ItemGroup>
-  <ItemGroup Condition=" '$(IsFullFramework)' == 'true' ">
-    <Reference Include="System.Configuration" />
-    <Reference Include="System.Runtime.Remoting" />
-  </ItemGroup>
-  <ItemGroup Condition="'$(ReferencedPlatform)' == 'net40'">
-    <Reference Include="System.Xaml" />
-  </ItemGroup>
-  <ItemGroup Condition="'$(ReferencedPlatform)' == 'net45'">
-    <Reference Include="System.Xaml" />
-  </ItemGroup>
+    <ProjectReference Include="..\Microsoft.Scripting.Metadata\Microsoft.Scripting.Metadata.csproj" />
```

Comment:
Can you add a `Condition=" $(Features.Contains('FEATURE_METADATA_READER')) "` to this reference?